src/components/Workspace.jsx

```diff
@@ -95,7 +112,10 @@ class Workspace extends React.Component {
       '_handleRequestedLineFocused',
       '_handleNotificationDismissed',
       '_handleExportGist',
+      '_storeDividerRef',
+      '_storeColumnRef',
     );
+    this.columnRefs = new Map([[1, null], [2, null], [3, null]]);
```

Comment:
It feels weird that we have to initialize a map with three elements, even though it will only ever hold two. How difficult would it be to be more flexible about that, or at least push that concern into the resizer module?